build.gradle

```diff
@@ -71,17 +71,13 @@ dependencies {
     compile group: 'org.scala-lang', name: 'scala-compiler', version: scalaVersion
     compile group: 'org.scala-lang.modules', name: 'scala-java8-compat_2.12', version: scalaJava8CompatVersion
     compile group: 'com.jsuereth', name: 'scala-arm_2.12', version: '2.0'
-    compile group: 'org.scala-lang.modules', name: 'scala-java8-compat_2.12', version: '0.9.1'
```

Comment:
Please keep the version in the properties file as-is

Reply:
done